docs/source/examples/predict-remaining-useful-life/example.ipynb

```diff
@@ -408,6 +408,59 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "lt.plot.distribution();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that most engines last between 118 to 287 cycles before needing maintenance. With these labels, we can train a machine learning model to predict which range the RUL will fall in."
```

Comment:
i think this paragraph is out of place. maybe it i supposed to be on the distribution plot a couple cells above?

Reply:
Moved cells

Reply:
On read the docs, not sure why the distribution plot doesn't show the discrete version. I've tried refreshing, wiping environment, and reactivating version. Locally, works fine.